templates/ts-apollo-fullstack/server/src/schema/schema.graphql

```diff
@@ -7,23 +7,53 @@ type Comment {
   description: String
 
   """@manyToOne field: 'comments', key: 'noteId'"""
-  note: Note
+  note(filter: NoteFilterUniqueInput): Note
+}
+
+input CommentFilterInput {
+  id: IDInput
+  text: StringInput
+  description: StringInput
+  and: [CommentFilterInput]
+  or: [CommentFilterInput]
+  not: [CommentFilterInput]
+}
+
+input CommentFilterUniqueInput {
+  id: ID!
 }
 
 input CommentInput {
   id: ID
   text: String
   description: String
-  noteId: ID
+}
+
+type CommentListResult {
+  items: [Comment]!
+  nextCursor: String
+}
+
+input IDInput {
```

Comment:
Static input type for each scalar

Reply:
This is where our previous print mechanism were much better. 
Printing this across large schema is minor issue but stands up when seeing this for the first time